schematic/utils/general.py

```diff
@@ -129,40 +129,19 @@ def calculate_datetime(
     return date_time_result
 
 
-def check_synapse_cache_size(
-    directory: str = "/root/.synapseCache",
-) -> float:
-    """use du --sh command to calculate size of .synapseCache.
+def check_synapse_cache_size(directory: str = "/root/.synapseCache") -> float:
```

Comment:
Need to potentially look into the the default values

Reply:
I'm not actually sure... if we should refactor this in this PR, but if we were, I probably would set the default value to `None` and then see if `/root/.synapseCache` exists.  If not... then default it to the system default's synapse cache, but I'm not sure if this will be an issue.

Reply:
I've seen two versions of the default: `~/.synapseCache` is the default in the `.synapseConfig` and then in `SynapseStorage.__init__` it says it defaults to `None`